mne/epochs.py

```diff
@@ -757,7 +757,31 @@ def drop_bad_epochs(self):
             disk. To avoid reading epochs form disk multiple times, initialize
             Epochs object with preload=True.
 
+        Parameters
+        ----------
+        reject : dict | None
+            Rejection parameters based on peak-to-peak amplitude.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg'.
+            If reject is None then no rejection is done. Example::
+
+                reject = dict(grad=4000e-13, # T / m (gradiometers)
+                              mag=4e-12, # T (magnetometers)
+                              eeg=40e-6, # uV (EEG channels)
+                              eog=250e-6 # uV (EOG channels)
+                              )
+
+        flat : dict | None
+            Rejection parameters based on flatness of signal.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg', and values
+            are floats that set the minimum acceptable peak-to-peak amplitude.
+            If flat is None then no rejection is done.
+
         """
+        if reject is not None:
```

Comment:
to piggyback on @agramfort's comment, i think preload should be checked first, if preload is True, we should check the reject parameter against the Epochs reject attribute to see if it is different (same should be done for flat). If they are not different, we should throw an error or warning that parameters are the same and that rejection has already been done, otherwise, it's run a second time.